opm/material/fluidmatrixinteractions/EclDefaultMaterial.hpp

```diff
@@ -170,30 +173,36 @@ class EclDefaultMaterial : public TraitsT
      * Hysteresis parameters for oil-water
      * @see EclHysteresisTwoPhaseLawParams::pcSwMdc(...)
      * @see EclHysteresisTwoPhaseLawParams::krnSwMdc(...)
+     * @see EclHysteresisTwoPhaseLawParams::deltaSwImbKrn(...)
      * \param params Parameters
      */
     template <class Evaluation>
     static void setOilWaterHysteresisParams(const Evaluation& pcSwMdc,
                                             const Evaluation& krnSwMdc,
+                                            const Evaluation& deltaSwImbKrn,
                                             Params& params)
     {
         params.oilWaterParams().setPcSwMdc(pcSwMdc);
         params.oilWaterParams().setKrnSwMdc(krnSwMdc);
+        params.oilWaterParams().setDeltaSwImbKrn(deltaSwImbKrn);
```

Comment:
I guess this is not really necessary because is recalculated if you use `params.oilWaterParams().update()` instead of the individual setter methods. You will need the reversal saturation for the relperm of the wetting fluid, though. the reason why the saturation delta is separately stored by the material parameter objects is that inverting the relperm curve is quite expensive. (since it's been a while since I wrote this code, take that statement with a grain of salt.)